graphqa/util.py

```diff
@@ -0,0 +1,168 @@
+from collections import Counter
+import numpy as np
+import string
+import re
+import tensorflow as tf
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+UNK = "<unk>"
+SOS = "<sos>"
+EOS = "<eos>"
+SPACE = "<space>"
+special_tokens = [UNK, SOS, EOS, SPACE]
+
+UNK_ID = special_tokens.index(UNK)
+SOS_ID = special_tokens.index(SOS)
+EOS_ID = special_tokens.index(EOS)
+
+CYPHER_PUNCTUATION = "()[]-=\"',.;:?"
+ENGLISH_PUNCTUATION = '!"#$%&()*+,-./:;=?@[\\]^_`{|}~'
+
+
+def load_vocab(args):
+    """
+    Load the tokens from the file constructed by "build_vocab" function in
+    build_data.py.
+    """
```

Comment:
Perhaps should add docstring for `args`, like what kwargs are expecting from args.

Reply:
Docstring added for "build_vocab" function; also put a "epilog" placeholder in args.py (will work on it later).